frontend/src/components/messages/NewRequestDialog.vue

```diff
@@ -75,51 +116,162 @@
                         :model-value="someFacilitiesSelected"></v-checkbox-btn>
                     </template>
                   </v-list-item>
-
                   <v-divider class="mt-2"></v-divider>
                 </template>
               </v-select>
             </v-col>
           </v-row>
-          <v-row no-gutters align="center">
-            <div class="mr-8">
-              <AppLabel variant="modal">Supporting documents (optional):</AppLabel>
-            </div>
-            <AppDocumentUpload v-model="documentsToUpload" entityName="ofm_assistance_requests" :loading="isLoading" @validateDocumentsToUpload="validateDocumentsToUpload"></AppDocumentUpload>
+          <v-row v-if="isAnAccountMaintenanceRequest && showFacility" no-gutters class="pb-6">
+            <v-col class="v-col-12 v-col-md-3 v-col-xl-1 pt-3 mt-0" />
+            <v-col class="v-col-12 v-col-md-9 v-col-xl-11 mt-0">
+              <span class="facility-tip">&nbsp;Submit a separate request for each facility</span>
+            </v-col>
           </v-row>
-          <v-row no-gutters class="mt-2">
+          <template v-if="isOrgPhoneEmailChecked">
+            <v-row no-gutters>
+              <v-col class="v-col-12 v-col-md-3 v-col-xl-3 pt-1 mb-0">
+                <AppLabel variant="modal">Organization phone (land):</AppLabel>
+              </v-col>
+              <v-col class="v-col-12 v-col-md-9 v-col-xl-9 mb-0">
+                <v-text-field
+                  v-model.trim="organizationModel.phoneLandline"
+                  :placeholder="PHONE_FORMAT"
+                  maxlength="12"
+                  variant="outlined"
+                  density="compact"
+                  :disabled="isLoading"></v-text-field>
+              </v-col>
+            </v-row>
+            <v-row no-gutters>
+              <v-col class="v-col-12 v-col-md-3 v-col-xl-3 pt-1 mb-0">
+                <AppLabel variant="modal">Organization phone (cell):</AppLabel>
+              </v-col>
+              <v-col class="v-col-12 v-col-md-9 v-col-xl-9 mb-0">
+                <v-text-field
+                  v-model.trim="organizationModel.phoneCell"
```

Comment:
Do we need to validate the phone format? (i.e.: rules="[...rules.required, rules.phone]")

Reply:
we do, that was added a few commits ago.